serverless/context.go

```diff
@@ -0,0 +1,11 @@
+package serverless
+
+// Context sfn handler context
+type Context interface {
+	// Data incoming data
+	Data() []byte
+	// Tag incoming tag
+	Tag() uint32
+	// Write write data to zipper
+	Write(tag uint32, data []byte) error
```

Comment:
How about defining a general `Writer` interface, which can be inherited by  both `Source` and `Context`?